.github/actions/save-logs/action.yaml

```diff
@@ -10,7 +10,7 @@ runs:
         sudo chown $USER /tmp/uds-*.log || echo ""
       shell: bash
 
-    - uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
+    - uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3 # v4.3.1
       with:
         name: debug-log
```

Comment:
We will need to add an input to ensure these names are unique across the parallel tests that run, each job can call this with something like pkg + flavor appended.

Reply:
Resolved in https://github.com/defenseunicorns/uds-core/pull/79/commits/622cb6bb505a3f2589132a70a4fb1661984d3290